tools/nl/svindex_differ/differ.py

```diff
@@ -46,13 +54,52 @@
 _FILE_PATTERN_EMBEDDINGS = r'embeddings_.*_\d{4}_\d{2}_\d{2}_\d{2}_\d{2}_\d{2}\.csv'
 _FILE_PATTERN_FINETUNED_EMBEDDINGS = r'embeddings_.*_\d{4}_\d{2}_\d{2}_\d{2}_\d{2}_\d{2}\.ft.*\.csv'
 
+AUTOPUSH_KEY = os.environ.get('AUTOPUSH_KEY')
+assert AUTOPUSH_KEY
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
+
+
+def _get_sv_names(sv_dcids):
+  """Get stat var names by making a mixer call"""
+  headers = {'Content-Type': 'application/json'}
+  headers['x-api-key'] = AUTOPUSH_KEY
+  resp = requests.post(_PROPERTY_URL,
+                       json={
+                           'nodes': sv_dcids,
+                           'property': '->name'
+                       },
+                       headers=headers).json()
+  result = {}
+  for node, node_arcs in resp.get('data', {}).items():
+    for v in node_arcs.get('arcs', {}).get('name', {}).get('nodes', []):
+      if 'value' in v:
+        result[node] = (v['value'])
+        break
+  return result
+
 
 def _prune(res):
-  result = []
+  svs = []
+  sv_info = {}
   for i in range(len(res['SV'])):
-    if i < _NUM_SVS and res['CosineScore'][i] >= _SV_THRESHOLD:
-      result.append(res['SV'][i])
-  return result
+    score = res['CosineScore'][i]
+    if i < _NUM_SVS and score >= _SV_THRESHOLD:
+      sv = res['SV'][i]
+      svs.append(sv)
+      sv_info[sv] = {
+          'sv': sv,
+          'rank': i + 1,
+          'score': score,
+          'sentence_scores': res['SV_to_Sentences'].get(sv, [])
+      }
+  return svs, sv_info
+
+
+def _get_file_name():
+  """Get the file name to use"""
+  username = pwd.getpwuid(os.getuid()).pw_name
+  date = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
```

Comment:
should the file name contain the index name too?  (medium_ft)

Reply:
nit: Also end with `.html`

Reply:
done